### ISSUE TITLE
PS Move Controller is moving near and far, acording with PS Eye, not the HMD (RiftCat - Cardboard)

### ISSUE THREAD

**Issue by andregm3** [cboulay/PSMoveService#118]
I'm using Cardboard + RiftCat with no Position Tracking HMD, in Steam VR the controller is moving like the PS EYE was the HMD, if i put the PS Move closer to the PS EYE, the HMD image of the Move goes bigger according to the PS EYE, not with the Cardboard view.

**Comment by zelmon64** [comment]
Can you see where the virtual PSEye is? If it's behind you then this behaviour would make sense. I use this with RiftCat VRidge as well and I have no problems.

**Comment by andregm3** [comment]
how to put the virtual PS Eye on the same place as the HMD ?

**Comment by zelmon64** [comment]
That's a point. I think if the tracker pose calibration fails the default position of the PSEye is the HMD origin. try redoing calibration.

**Comment by andregm3** [comment]
the attach to HMD calibration never works but i got some results with Calibration Mat

**Comment by zelmon64** [comment]
Yeah the HMD method wouldn't work since it doesn't have positional tracking like you said. Can you now see the PSEye in VR and does the PSMove move correctly relative to it?

**Comment by andregm3** [comment]
The PS Eye will show in the Steam VR envioriment ? i did the MAT but not very well located but at least the result was different from the HMD attached.

RiftCat also has a recenter button option.

![calibrationmat](https://cloud.githubusercontent.com/assets/4154732/16838887/3711f6b0-49a2-11e6-8a1a-6b5573eb1dd0.jpg)

**Comment by zelmon64** [comment]
Yes you should see a 3d model of the PSEye in SteamVR. In the calibration tool the PSEye is represented by the origin of the tracker volume as can be seen to the right of [this image](https://raw.githubusercontent.com/cboulay/PSMoveService/master/docs/TrackerPose/TestTracking.jpg)(there are two there).

Do you have the mat positioned correctly as in flat with 4 and 5 to the front? You may also want to double check the colour masks.

**Comment by andregm3** [comment]
yes i got it, hmm 4 and 5 the front i tought it was the back

**Comment by andregm3** [comment]
https://www.youtube.com/watch?v=FFJqLH_dkRw

**Comment by zelmon64** [comment]
Well that's the orientation I've always used and it works for me. I can't actually see anywhere that explicitly  state how the mat should be orientated but if 4 and 5 are to the back then the numbers would appear upside down and that wouldn't make sense to me. 

I believe there should be some new videos on the way soon.
